### PR TITLE
Fix conflicting config file ownership. Contributes to JB#52809

### PR DESCRIPTION
**Pull request by pvuorela**
/etc/ofono/phonesim.conf owned both by main package and the config subpackage. Suppose it should be config. Pulled in by the parent package anyhow.

@monich @tomin1 @spiiroin

### CODE REVIEW

File: rpm/phonesim.spec [hunk 1]
```diff
@@ -48,24 +36,13 @@ FallbackTimeservers.
 %prep
 %setup -q -n %{name}-%{version}
 
-# >> setup
-# << setup
-
 %build
-# >> build pre
-# << build pre
```
Comment (Tomin1) [inline rpm/phonesim.spec]: I would skip this empty line too. Maybe also the next empty line as well.

File: rpm/phonesim.spec [hunk 2]
```diff
@@ -1,18 +1,8 @@
...
-
-# >> macros
-# << macros
-
 Summary:    Phone simulator.
 Version:    1.19.0
 Release:    1
-Group:      Applications/System
-License:    LGPL v2.1
+License:    LGPLv2
```
Comment (Tomin1) [inline rpm/phonesim.spec]: Is this correct? `COPYING` is for `GPLv2` and headers seem to indicate that too. (Also funny enough, headers talk about `LICENSE.GPL` and not `COPYING`).

File: rpm/phonesim.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/phonesim.spec]: Sure.

File: rpm/phonesim.spec (hunk 2 above)
Comment (pvuorela) [reply rpm/phonesim.spec]: Oh indeed. Changed to GPLv2

**Review by Tomin1 — COMMENTED** [review]
One more thing: I looked at what the packages now contain and I noticed that `/usr/share/phonesim` directory is not owned by phonesim while I would assume it to be part of the main package. Otherwise looks good to me.

**Comment by pvuorela** [comment]
Added the directory too

**Review by Tomin1 — APPROVED** [review]

**Pull request merged**